model-archiver/setup.py

```diff
@@ -43,20 +43,23 @@ def pypi_description():
         return df.read()
 
 
-def detect_packaging_tool_version():
+def detect_model_archiver_version():
+    if "--release" in sys.argv:
+        sys.argv.remove("--release")
+        # pylint: disable = relative-import
+        return model_archiver.__version__.strip() + '.' + str(date.today()).replace('-', '')
```

Comment:
We don't need this for model-archiver. It might just be enough to remove 'b' tag from the setup.py.

Reply:
With this change, we can also create a model-archiver-release project on codebuild.

Reply:
Ok.